README.md

```diff
@@ -0,0 +1,171 @@
+
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Final Capstone Front End ](#-final-capstone-front-end-)
+  - [💻 Link to Back End ](#-link-to-back-end-)
+  - [💻 Link to Kanban board information ](#-link-to-kanban-board-information-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ (OPTIONAL) ](#-faq-optional-)
+  - [📝 License ](#-license-)
+# 📖 Final Capstone Front End <a name="about-project"></a>
+
+**Final Capstone Front End** The front-end component of this final capstone project is developed using React.js, designed to provide an intuitive user interface for managing creation of doctors to rent and make reservations. It communicates with the back-end API built with Ruby on Rails.
+
+## 💻 Link to Back End <a name="link-to-back-end"></a>
+
+[Link to Back End](https://github.com/EvansSnave/final-capstone-project-back-end)
+
+<p align="right"\>(<a href="#readme-top"\>back to top</a>)</p>
+
+## 💻 Link to Kanban board information <a name="built-with"></a>
+
+[Link to Kanban Board](https://github.com/EvansSnave/final-capstone-project-back-end/projects/1)
+
+[Initial state](https://github.com/EvansSnave/final-capstone-project-front-end/issues/17)
+
+This project was completed by three Team members:
+
+- Kevin Cruz
+- Fisayo Michael
+
+<p align="right"\>(<a href="#readme-top"\>back to top</a>)</p>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Technologies</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **React.js Components**
+- **API Communication with Backend**
+- **User-Friendly Interface**
+- **Responsive Design**
+- **State Management**
+
+<p align="right"\>(<a href="#readme-top"\>back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+Before you begin, make sure you have the following prerequisites installed on your system:
+
+- Node.js: You need Node.js to run the React.js application.
+- npm: npm is used to manage packages in your React project.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+sh <br>
+cd my-folder <br>
+git clone https://github.com/EvansSnave/final-capstone-project-front-end.git
+
+### Install
+
+Install this project with:
+
+```bash
+npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```bash
+npm run dev
+```
```

Comment:
- [ ] According to your `package.json` this command is not going to work:

![image](https://github.com/EvansSnave/final-capstone-project-front-end/assets/57421823/bfb665e0-6ede-47b2-8c8f-b925b3d23201)

The correct command is `npm start`.  Kindly correct these instructions so your users can run your app correctly. 💪